dist/nginx.conf

```diff
@@ -60,13 +60,11 @@ server {
 
   location ~ ^/(emoji|packs|system/accounts/avatars|system/media_attachments/files) {
     add_header Cache-Control "public, max-age=31536000, immutable";
-    add_header Strict-Transport-Security "max-age=31536000" always;
     try_files $uri @proxy;
   }
 
   location /sw.js {
     add_header Cache-Control "public, max-age=0";
-    add_header Strict-Transport-Security "max-age=31536000" always;
```

Comment:
I believe these should be kept (or rather, changed to `max-age=63072000; includeSubDomains` for consistency, along with the one on line 55), and the one that needs to be removed is the one in the `location @proxy` block (on line 93).